src/Execution/ExecutionContextBuilder.php

```diff
@@ -67,12 +69,25 @@ public function buildContext(
             throw new ExecutionException('Must provide an operation.');
         }
 
+        $variableValues = [];
+
+        /** @var OperationDefinitionNode $operation */
+        if ($operation) {
+            $coercedVariableValues = (new ValuesHelper())->coerceVariableValues(
+                $schema,
+                $operation->getVariableDefinitions(),
+                $rawVariableValues
+            );
+
+            $variableValues = $coercedVariableValues['coerced'];
```

Comment:
I would either return an object or use a destructor statement to assign the variables.

Reply:
Yeah, I also think it's better to return an object.